invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -182,49 +186,49 @@ const LogoUploader = (props) => {
 };
 
 const DangerZone = ({ community, onError }) => (
-    <Segment color="red" tertiary className="rel-mt-2">
-      <Header as="h2" color="red">
-        {i18next.t("Danger zone")}
-      </Header>
-      <Grid>
-        <Grid.Column width="12">
-          <Header as="h4">{i18next.t("Rename community")}</Header>
-          <p>
-            {i18next.t(
-              "Renaming your community can have unintended side effects."
-            )}
-          </p>
-        </Grid.Column>
-        <Grid.Column floated="right" width="4">
-          <RenameCommunityButton community={community} />
-        </Grid.Column>
-        <Grid.Column floated="left" width="12">
-          <Header as="h4">{i18next.t("Delete community")}</Header>
-          <p>
-            {i18next.t(
-              "Once deleted, it will be gone forever. Please be certain."
-            )}
-          </p>
-        </Grid.Column>
-        <Grid.Column floated="right" width="4">
-          <DeleteButton
-            community={community}
-            label={i18next.t("Delete community")}
-            redirectURL="/communities"
-            confirmationMessage={
-              <h3>
-                {i18next.t("Are you sure you want to delete this community?")}
-              </h3>
-            }
-            onDelete={() => {
-              const client = new CommunitiesApiClient();
-              return client.delete(community.id);
-            }}
-            onError={onError}
-          />
-        </Grid.Column>
-      </Grid>
-    </Segment>
+  <Segment color="red" tertiary className="rel-mt-2">
+    <Header as="h2" color="red">
+      {i18next.t("Danger zone")}
+    </Header>
+    <Grid>
+      <Grid.Column width="12">
+        <Header as="h4">{i18next.t("Rename community")}</Header>
+        <p>
+          {i18next.t(
+            "Renaming your community can have unintended side effects."
+          )}
+        </p>
+      </Grid.Column>
+      <Grid.Column floated="right" width="4">
+        <RenameCommunityButton community={community} />
+      </Grid.Column>
+      <Grid.Column floated="left" width="12">
+        <Header as="h4">{i18next.t("Delete community")}</Header>
+        <p>
+          {i18next.t(
+            "Once deleted, it will be gone forever. Please be certain."
+          )}
+        </p>
+      </Grid.Column>
+      <Grid.Column floated="right" width="4">
+        <DeleteButton
+          community={community}
+          label={i18next.t("Delete community")}
+          redirectURL="/communities"
+          confirmationMessage={
+            <h3>
+              {i18next.t("Are you sure you want to delete this community?")}
+            </h3>
+          }
+          onDelete={async () => {
+            const client = new CommunitiesApiClient();
+            await client.delete(community.id);
```

Comment:
what happens after deleting th community ?

Reply:
it will reload the page